docs/docs/examples/evaluation/batch_eval.ipynb

```diff
@@ -19,12 +19,12 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    print(eval_results["faithfulness"][0].dict().keys())
</pre><p>main change here seems to be switching to faithfulness from correctness, why is that </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
correctness is commented out earlier in the notebook.